src/OSPSuite.Infrastructure.Import/Core/DataSource.cs

```diff
@@ -134,10 +134,10 @@ public ParseErrors AddSheets(Cache<string, DataSheet> dataSheets, ColumnInfoCach
          return errors;
       }
 
-      public void SetMappings(string fileName, IEnumerable<MetaDataMappingConverter> mappings)
+      public void SetMappings(string fileName, IList<MetaDataMappingConverter> mappings)
       {
          _configuration.FileName = fileName;
-         _mappings = mappings;
+         _mappings = (IReadOnlyList<MetaDataMappingConverter>)mappings;
```

Comment:
especially because you are casting back to IReadOnlyList... what? :)

Reply:
true, corrected